src/EventSubscriber/UserCreationSubscriber.php

```diff
@@ -133,10 +133,13 @@ private function addCreatorClient(User $user): void
 
     private function addCreators(?User $user): void
     {
-        if ($user->getSubjectBeneficiaire() || $user->getSubjectMembre()) {
+        if ($user->getSubjectBeneficiaire()) {
             $this->addCreatorUser($user);
             $this->addCreatorRelay($user);
             $this->addCreatorClient($user);
+        } elseif ($user->getSubjectMembre()) {
+            $this->addCreatorUser($user);
```

Comment:
```
 private function addCreators(?User $user): void
    {
        if ($user->isAdministrateur() || $user->isSuperAdmin()) {
            return;
        }

        if ($user->isBeneficiaire()) {
            $this->addCreatorRelay($user);
        }

        $this->addCreatorUser($user);
        $this->addCreatorClient($user);
    }

```

Reply:
Ou encoree 
`private function addCreators(?User $user): void
    {
        if (!$user->isBeneficiaire() && !$user->isMembre()) {
            return;
        }

        if ($user->isBeneficiaire()) {
            $this->addCreatorRelay($user);
        }

        $this->addCreatorUser($user);
        $this->addCreatorClient($user);
    }`